content/upcoming/enry.md

```diff
@@ -0,0 +1,181 @@
+---
+author: mcarmonaa
+date: 2017-07-03
+draft: true
+title: "enry: detecting your accent"
```

Comment:
I must say that I don't like the title too much... accents are variants inside a natural language and we could call that to some different coding conventions inside the same language (like K&R braces vs GNU, etc) while enry detect languages.